apps/meteor/client/components/ImageGallery/ImageGallery.tsx

```diff
@@ -32,10 +32,8 @@ const swiperStyle = css`
 		background-color: var(--rcx-color-surface-overlay, rgba(0, 0, 0, 0.6));
 	}
 
-	.rcx-swiper-close-button,
-	.rcx-swiper-prev-button,
-	.rcx-swiper-next-button {
-		color: var(--rcx-color-font-pure-white, #ffffff) !important;
+	.swiper-slide {
+		padding: 9rem 3.75rem 6rem;
```

Comment:
these hardcoded padding values look odd... is it like this by design? couldn't we follow our pattern of x4 values?

Reply:
I agree we should se the x pattern and applied the same.
I put those values to not let the image coincide with the buttons and the top toolbar when the window is resized.